src/test/java/com/javachallengers/simpson/model/dto/SimpsonCharacterRequestDTOTest.java

```diff
@@ -0,0 +1,23 @@
+package com.javachallengers.simpson.model.dto;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+/**
+ * Unit tests for {@link SimpsonCharacterRequestDTO}
+ */
+@Tag("UnitTest")
+class SimpsonCharacterRequestDTOTest {
+	@Test
+	void allArgsConstructor() {
+		String name = "Homer";
```

Comment:
In my opinion, it's overkill to test DTOs because we are already testing it in the service. Would you be ok with removing it? What are your thoughts?

Reply:
I can remove it. Yes it is overkill but I'm used to do this because some tools like sonarqube complains about lack of unit tests on these type of classes.

Reply:
Oh, bad configuration on Sonar brings us a lot of headaches! We should avoid to follow tools numbers and implement what has value to the software.

Reply:
Ok I'll remove it.

Reply:
Removed.